industry_restaurant/data/account_cash_rounding.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="account_cash_rounding_1" model="account.cash.rounding">
```

Comment:
Don't do that, it is supposed to be specific to your currency and/or country, you don't need it.